.buildkite/commands/release-upload.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+
+"$(dirname "${BASH_SOURCE[0]}")/shared_setup.sh"
+
+echo "--- :arrow_down: Downloading Artifacts"
+ARTIFACTS_DIR='artifacts' # Defined in Fastlane, see ARTIFACTS_FOLDER
+STEP=testflight_build
+buildkite-agent artifact download "$ARTIFACTS_DIR/*.ipa" . --step $STEP
+buildkite-agent artifact download "$ARTIFACTS_DIR/*.zip" . --step $STEP
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
+
+echo "--- :testflight: Uploading to TestFlight"
+bundle exec fastlane upload_app_store_connect_build_to_testflight
```

Comment:
It came natural to call each lane individually here in the build script. Of course, we could have a single lane here that calls the other lanes.

Maybe it would be better from the point of view of making it simple to run the uploads from a local machine in the those rare cases when they can't run in CI.